extensions/gitpod/src/extension.ts

```diff
@@ -69,6 +69,32 @@ export async function activate(context: vscode.ExtensionContext) {
 	const gitpodApi = workspaceInfoResponse.getGitpodApi()!;
 	const workspaceContextUrl = vscode.Uri.parse(workspaceInfoResponse.getWorkspaceContextUrl());
 
+	//#region LeapIDE config update
+	(async () => {
+		// update `name` to `value` in "User settings" if user didn't explicitly set/modify it
+		function updateIfDefault(config: vscode.WorkspaceConfiguration, name: string, value: any) {
+			if (config.get(name) === config.inspect(name)?.defaultValue) {
+				config.update(name, value, vscode.ConfigurationTarget.Global);
+			}
+		}
+		let config = vscode.workspace.getConfiguration();
```

Comment:
can use `const` (optional)

Reply:
IIRC, `config.update` call did not work when `config` was declared as `const`. I'll double-check.